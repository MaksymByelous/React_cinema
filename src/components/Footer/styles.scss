footer {
    padding: 10px;
    text-align: center;
    background: #474e5a;
    color:  white;
}