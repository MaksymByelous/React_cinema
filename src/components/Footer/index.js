import React from 'react';
import './styles.scss';

const Footer = () => (
    <footer>Movies database, 2017</footer>
);

export default Footer;
