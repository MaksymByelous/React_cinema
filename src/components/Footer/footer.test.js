import React from 'react';
import dom from 'react-test-renderer';
import Footer from './';

const renderTree = dom.create(
    <Footer />
).toJSON();

test(
    'footer should correspond to snapshot', () => {
        expect(renderTree).toMatchSnapshot();
    }
);
