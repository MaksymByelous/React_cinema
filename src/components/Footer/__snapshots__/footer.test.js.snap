// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`footer should correspond to snapshot 1`] = `
<footer>
  Movies database, 2017
</footer>
`;
